common/lib/xmodule/xmodule/css/capa/display.scss

```diff
@@ -414,6 +414,12 @@ div.problem {
     }
   }
 
+  div.inline {
+    > span {
```

Comment:
@sarina I went looking at the CSS; I can't find any reference to a class called "wrapper-inline". I couldn't find it in the output of the SASS either, nor in the output HTML. I'm somewhat confused - what exactly are you referring to?